app/src/main/res/layout/activity_maps.xml

```diff
@@ -51,21 +51,24 @@
 
     <Button
         android:id="@+id/locationButton"
```

Comment:
run the formatter of android studio on both files to respect the style convention. Otherwise, looks perfect, great job!